gmso/core/topology.py

```diff
@@ -625,6 +625,143 @@ def is_typed(self, updated=False):
             self._typed = False
         return self._typed
 
+    def is_fully_typed(self, updated=False, group="topology"):
+        """Check if the topology certain group of objects of topology is fully or partially typed
+
+        Parameters
+        ----------
+        updated : bool, optional, default=False
+            If False, will update the atom type and connection type list of the
+            Topology before the checking step.
+        group : str, optional, default='topology'
+            Specific object to be checked. Options include:
+            'topology'  : check for status of the topology object
+            'sites'     : check for status of all topology.sites
+            'bonds'     : check for status of all topology.bonds
+            'angles'    : check for status of all topology.angles
+            'dihedrals' : check for status of all topology.dihedrals
+            'impropers' : check for status of all topology.impropers
+        full : bool, optional, default=True
```

Comment:
We dont actually have this parameter in the function definition above. Do we actually need this option? This makes the name of the function a bit confusing. If `is_fully_typed` had `full` set to `False`, that essentially changes the behavior of the function all together.

I think it would be reasonable to remove the `full` option all together. And if `is_fully_typed` returns false, then the user can call `get_untyped` method to discover what is missing.